goa_header_methods/generator.go

```diff
@@ -0,0 +1,107 @@
+package goaheadermethods
```

Comment:
Thanks! I renamed the package to `goasupport` to match with the existing `gormsupport` one.